apps/electron-app/src/main/browser/tab-lifecycle-manager.ts

```
@@ -0,0 +1,271 @@
+
+    if (hasChanges) {
+      this.eventBus.emit("tab-lifecycle:reordered", { tabs: sortedTabs });
+    }
+  }
+
+  private trackTabCreation(targetUrl: string): void {
+    const mainWindows = this.browser
+      .getAllWindows()
+      .filter((w: any) =>
+        w &&
+        w.webContents &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)